app/jobs/create_individual_groups_for_all_students_job.rb

```diff
@@ -38,6 +38,10 @@ def perform(assignment)
 
       # Generate the permissions file for all valid groups
       Repository::SubversionRepository.__generate_authz_file
+      m_logger = MarkusLogger.instance
+      m_logger.log("Creating all individual groups completed",
+                   MarkusLogger::INFO)
+      puts "Creating all groups complete"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.